### DISCUSSION TITLE
Tell me any of your favorite songs!

### DISCUSSION THREAD

**Discussion by chamakkon** [NAIST-SE/AttractingContributors#1196] [2024 Attraction Contributions]
Please make a contribution to my repo!!!
https://github.com/chamakkon/Fav-Songs
I'm currently 4 PR short of 10.  I'll also make a contribution to your project too.

**Comment by marshr00m** [comment]
I sent a pull request to your repo. I would appreciate any additions to my list!
https://github.com/marshr00m/AttractingContributors-Where-would-you-like-to-travel

**Comment by wangzaixiaoliunai** [comment]
I have created a PR.
Could you please contribute to mine?
https://github.com/wangzaixiaoliunai/AttractingContributors-musical_instruments.git
